[Bug] Java CI: DensukeWriteServiceTest と WaitlistPromotionServiceTest が main で常時失敗 (14件)

## バグの概要
`main` ブランチの `Java CI with Gradle` が継続的に赤い。`./gradlew test` で 1154 件中 14 件のテストが失敗している。

### 失敗テスト内訳

**DensukeWriteServiceTest（2件・`UnnecessaryStubbingException`）**
- `dirty=trueの参加者がいない場合は何も書き込まない`
- `BYE(matchNumber=null)のみdirtyの場合はプレイヤーの書き込みが発生しない`

**WaitlistPromotionServiceTest（12件・`IllegalStateException` / `AssertionFailedError` / `WantedButNotInvoked` 等）**
- `promoteNextWaitlistedはLINE通知を送信しない`
- `expireOfferSuppressed: 通知抑制版オファー期限切れ処理 > OFFERED状態の参加者を渡すとDECLINEDに変更されExpireOfferResultが返る`
- `respondToOfferAll: 期限切れスキップ時の件数テスト > 一括承諾で期限切れがスキップされた場合、実処理件数が返される`
- `辞退起点の繰り上げLINE通知テスト > respondToOffer辞退時に繰り上げ先へLINE通知が送信される`
- `辞退起点の繰り上げLINE通知テスト > 辞退時に繰り上げ対象がいなければLINE通知は送信されない`
- `辞退起点の繰り上げLINE通知テスト > respondToOfferAll辞退時に各試合の繰り上げ先へLINE通知が送信される`
- `respondToOffer: 部分参加後の残りオファー通知テスト > 承諾後に残りOFFEREDがなければ通知は送信されない`
- `respondToOffer: 部分参加後の残りオファー通知テスト > 承諾時に管理者通知が送信される`
- `respondToOffer: 部分参加後の残りオファー通知テスト > 承諾後に残りOFFEREDがあれば通知が送信される`
- `respondToOfferAll: 一括応答テスト > 一括辞退で期限切れOFFEREDはスキップされる`
- `respondToOfferAll: 一括応答テスト > 全OFFEREDを一括承諾できる`
- `オファー期限切れ時の繰り上げLINE通知 > 繰り上げありの場合、sendWaitlistOfferNotificationが呼ばれる`

### 確認済みのこと
- main の最新 commit (`94f424b`) の CI run #26330634157 で再現
- PR #765（フロントエンドのみの変更）でも同じ失敗が再現したため、本Issueは独立した既存の基盤問題
- BUILD FAILED in 4m 8s（1154 tests completed, 14 failed）

## 再現手順
\`\`\`bash
cd karuta-tracker && ./gradlew test
\`\`\`
（または GitHub Actions の `Java CI with Gradle` ワークフロー）

## 期待される動作
全テスト PASS で CI が green になる。

## 実際の動作
14 件 FAILED で CI が red のまま。

--- a/karuta-tracker/src/test/java/com/karuta/matchtracker/service/DensukeWriteServiceTest.java
+++ b/karuta-tracker/src/test/java/com/karuta/matchtracker/service/DensukeWriteServiceTest.java
@@ -3,6 +3,7 @@ package com.karuta.matchtracker.service;
 import com.karuta.matchtracker.dto.DensukeWriteStatusDto;
 import com.karuta.matchtracker.entity.*;
 import com.karuta.matchtracker.repository.*;
+import com.karuta.matchtracker.util.JstDateTimeUtil;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
@@ -89,24 +90,30 @@ class DensukeWriteServiceTest {
     @Test
     @DisplayName("dirty=trueの参加者がいない場合は何も書き込まない")
     void testWriteToDensuke_noDirtyParticipants_skipsWrite() {
+        // 実装は writeToDensuke() で今月と来月のURLを取得するため、
+        // 現在の年月を基準にテストデータを構築する
+        LocalDate today = JstDateTimeUtil.today();
+        int currentYear = today.getYear();
+        int currentMonth = today.getMonthValue();
+
         DensukeUrl url = DensukeUrl.builder()
-                .id(1L).year(2026).month(4).organizationId(1L)
+                .id(1L).year(currentYear).month(currentMonth).organizationId(1L)
                 .url("https://densuke.biz/list?cd=test123").build();
         when(densukeUrlRepository.findByYearAndMonth(anyInt(), anyInt()))
                 .thenAnswer(inv -> {
                     int y = (int) inv.getArgument(0);
                     int m = (int) inv.getArgument(1);
-                    if (y == 2026 && m == 4) return List.of(url);
+                    if (y == currentYear && m == currentMonth) return List.of(url);
                     return List.of();
                 });
 
         PracticeSession session = PracticeSession.builder()
-                .id(10L).sessionDate(LocalDate.of(2026, 4, 1)).totalMatches(3).build();
+                .id(10L).sessionDate(today.withDayOfMonth(1)).totalMatches(3).build();
         when(practiceSessionRepository.findByYearAndMonthAndOrganizationId(anyInt(), anyInt(), eq(1L)))
                 .thenAnswer(inv -> {
                     int y = (int) inv.getArgument(0);
                     int m = (int) inv.getArgument(1);
-                    if (y == 2026 && m == 4) return List.of(session);
+                    if (y == currentYear && m == currentMonth) return List.of(session);
                     return Collections.emptyList();
                 });
 
@@ -165,24 +172,30 @@ class DensukeWriteServiceTest {
     @Test
     @DisplayName("BYE(matchNumber=null)のみdirtyの場合はプレイヤーの書き込みが発生しない")
     void testWriteToDensuke_byeOnlyDirty_skipsWrite() {
+        // 実装は writeToDensuke() で今月と来月のURLを取得するため、
+        // 現在の年月を基準にテストデータを構築する
+        LocalDate today = JstDateTimeUtil.today();
+        int currentYear = today.getYear();
+        int currentMonth = today.getMonthValue();
+
         DensukeUrl url = DensukeUrl.builder()
-                .id(1L).year(2026).month(4).organizationId(1L)
+                .id(1L).year(currentYear).month(currentMonth).organizationId(1L)
                 .url("https://densuke.biz/list?cd=test123").build();
         when(densukeUrlRepository.findByYearAndMonth(anyInt(), anyInt()))
                 .thenAnswer(inv -> {
                     int y = (int) inv.getArgument(0);
                     int m = (int) inv.getArgument(1);
-                    if (y == 2026 && m == 4) return List.of(url);
+                    if (y == currentYear && m == currentMonth) return List.of(url);
                     return List.of();
                 });
 
         PracticeSession session = PracticeSession.builder()
-                .id(10L).sessionDate(LocalDate.of(2026, 4, 1)).totalMatches(3).build();
+                .id(10L).sessionDate(today.withDayOfMonth(1)).totalMatches(3).build();
         when(practiceSessionRepository.findByYearAndMonthAndOrganizationId(anyInt(), anyInt(), eq(1L)))
                 .thenAnswer(inv -> {
                     int y = (int) inv.getArgument(0);
                     int m = (int) inv.getArgument(1);
-                    if (y == 2026 && m == 4) return List.of(session);
+                    if (y == currentYear && m == currentMonth) return List.of(session);
                     return Collections.emptyList();
                 });
 

--- a/karuta-tracker/src/test/java/com/karuta/matchtracker/service/DensukeWriteServiceTest.java
+++ b/karuta-tracker/src/test/java/com/karuta/matchtracker/service/DensukeWriteServiceTest.java
@@ -8,6 +8,7 @@ import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
+import org.mockito.MockedStatic;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
@@ -90,40 +91,44 @@ class DensukeWriteServiceTest {
     @Test
     @DisplayName("dirty=trueの参加者がいない場合は何も書き込まない")
     void testWriteToDensuke_noDirtyParticipants_skipsWrite() {
-        // 実装は writeToDensuke() で今月と来月のURLを取得するため、
-        // 現在の年月を基準にテストデータを構築する
-        LocalDate today = JstDateTimeUtil.today();
-        int currentYear = today.getYear();
-        int currentMonth = today.getMonthValue();
+        // 実装は writeToDensuke() で today() を呼んで今月と来月のURLを取得するため、
+        // テストセットアップとサービス呼び出しの間で月またぎが起きないよう today() を固定する
+        LocalDate fixedToday = LocalDate.of(2026, 5, 24);
+        int currentYear = fixedToday.getYear();
+        int currentMonth = fixedToday.getMonthValue();
 
-        DensukeUrl url = DensukeUrl.builder()
-                .id(1L).year(currentYear).month(currentMonth).organizationId(1L)
-                .url("https://densuke.biz/list?cd=test123").build();
-        when(densukeUrlRepository.findByYearAndMonth(anyInt(), anyInt()))
-                .thenAnswer(inv -> {
-                    int y = (int) inv.getArgument(0);
-                    int m = (int) inv.getArgument(1);
-                    if (y == currentYear && m == currentMonth) return List.of(url);
-                    return List.of();
-                });
+        try (MockedStatic<JstDateTimeUtil> jstMock = mockStatic(JstDateTimeUtil.class, CALLS_REAL_METHODS)) {
+            jstMock.when(JstDateTimeUtil::today).thenReturn(fixedToday);
 
-        PracticeSession session = PracticeSession.builder()
-                .id(10L).sessionDate(today.withDayOfMonth(1)).totalMatches(3).build();
-        when(practiceSessionRepository.findByYearAndMonthAndOrganizationId(anyInt(), anyInt(), eq(1L)))
-                .thenAnswer(inv -> {
-                    int y = (int) inv.getArgument(0);
-                    int m = (int) inv.getArgument(1);
-                    if (y == currentYear && m == currentMonth) return List.of(session);
-                    return Collections.emptyList();
-                });
+            DensukeUrl url = DensukeUrl.builder()
+                    .id(1L).year(currentYear).month(currentMonth).organizationId(1L)
+                    .url("https://densuke.biz/list?cd=test123").build();
+            when(densukeUrlRepository.findByYearAndMonth(anyInt(), anyInt()))
+                    .thenAnswer(inv -> {
+                        int y = (int) inv.getArgument(0);
+                        int m = (int) inv.getArgument(1);
+                        if (y == currentYear && m == currentMonth) return List.of(url);
+                        return List.of();
+                    });
 
-        when(practiceParticipantRepository.findDirtyForDensukeSync(any()))
-                .thenReturn(Collections.emptyList());
+            PracticeSession session = PracticeSession.builder()
+                    .id(10L).sessionDate(fixedToday.withDayOfMonth(1)).totalMatches(3).build();
+            when(practiceSessionRepository.findByYearAndMonthAndOrganizationId(anyInt(), anyInt(), eq(1L)))
+                    .thenAnswer(inv -> {
+                        int y = (int) inv.getArgument(0);
+                        int m = (int) inv.getArgument(1);
+                        if (y == currentYear && m == currentMonth) return List.of(session);
+                        return Collections.emptyList();
+                    });
 
-        densukeWriteService.writeToDensuke();
+            when(practiceParticipantRepository.findDirtyForDensukeSync(any()))
+                    .thenReturn(Collections.emptyList());
 
-        verify(densukeMemberMappingRepository, never()).findByDensukeUrlIdAndPlayerId(any(), any());
-        assertThat(densukeWriteService.getStatus(1L).getPendingCount()).isEqualTo(0);
+            densukeWriteService.writeToDensuke();
+
+            verify(densukeMemberMappingRepository, never()).findByDensukeUrlIdAndPlayerId(any(), any());
+            assertThat(densukeWriteService.getStatus(1L).getPendingCount()).isEqualTo(0);
+        }
     }
 
     @Test
@@ -172,42 +177,46 @@ class DensukeWriteServiceTest {
     @Test
     @DisplayName("BYE(matchNumber=null)のみdirtyの場合はプレイヤーの書き込みが発生しない")
     void testWriteToDensuke_byeOnlyDirty_skipsWrite() {
-        // 実装は writeToDensuke() で今月と来月のURLを取得するため、
-        // 現在の年月を基準にテストデータを構築する
-        LocalDate today = JstDateTimeUtil.today();
-        int currentYear = today.getYear();
-        int currentMonth = today.getMonthValue();
+        // 実装は writeToDensuke() で today() を呼んで今月と来月のURLを取得するため、
+        // テストセットアップとサービス呼び出しの間で月またぎが起きないよう today() を固定する
+        LocalDate fixedToday = LocalDate.of(2026, 5, 24);
+        int currentYear = fixedToday.getYear();
+        int currentMonth = fixedToday.getMonthValue();
 
-        DensukeUrl url = DensukeUrl.builder()
-                .id(1L).year(currentYear).month(currentMonth).organizationId(1L)
-                .url("https://densuke.biz/list?cd=test123").build();
-        when(densukeUrlRepository.findByYearAndMonth(anyInt(), anyInt()))
-                .thenAnswer(inv -> {
-                    int y = (int) inv.getArgument(0);
-                    int m = (int) inv.getArgument(1);
-                    if (y == currentYear && m == currentMonth) return List.of(url);
-                    return List.of();
-                });
+        try (MockedStatic<JstDateTimeUtil> jstMock = mockStatic(JstDateTimeUtil.class, CALLS_REAL_METHODS)) {
+            jstMock.when(JstDateTimeUtil::today).thenReturn(fixedToday);
 
-        PracticeSession session = PracticeSession.builder()
-                .id(10L).sessionDate(today.withDayOfMonth(1)).totalMatches(3).build();
-        when(practiceSessionRepository.findByYearAndMonthAndOrganizationId(anyInt(), anyInt(), eq(1L)))
-                .thenAnswer(inv -> {
-                    int y = (int) inv.getArgument(0);
-                    int m = (int) inv.getArgument(1);
-                    if (y == currentYear && m == currentMonth) return List.of(session);
-                    return Collections.emptyList();
-                });
+            DensukeUrl url = DensukeUrl.builder()
+                    .id(1L).year(currentYear).month(currentMonth).organizationId(1L)
+                    .url("https://densuke.biz/list?cd=test123").build();
+            when(densukeUrlRepository.findByYearAndMonth(anyInt(), anyInt()))
+                    .thenAnswer(inv -> {
+                        int y = (int) inv.getArgument(0);
+                        int m = (int) inv.getArgument(1);
+                        if (y == currentYear && m == currentMonth) return List.of(url);
+                        return List.of();
+                    });
 
-        // findDirtyForDensukeSync は matchNumber IS NOT NULL なのでBYEを返さない → 空
-        when(practiceParticipantRepository.findDirtyForDensukeSync(any()))
-                .thenReturn(Collections.emptyList());
+            PracticeSession session = PracticeSession.builder()
+                    .id(10L).sessionDate(fixedToday.withDayOfMonth(1)).totalMatches(3).build();
+            when(practiceSessionRepository.findByYearAndMonthAndOrganizationId(anyInt(), anyInt(), eq(1L)))
+                    .thenAnswer(inv -> {
+                        int y = (int) inv.getArgument(0);
+                        int m = (int) inv.getArgument(1);
+                        if (y == currentYear && m == currentMonth) return List.of(session);
+                        return Collections.emptyList();
+                    });
 
-        densukeWriteService.writeToDensuke();
+            // findDirtyForDensukeSync は matchNumber IS NOT NULL なのでBYEを返さない → 空
+            when(practiceParticipantRepository.findDirtyForDensukeSync(any()))
+                    .thenReturn(Collections.emptyList());
 
-        // BYEのみdirtyの場合、書き込み処理に進まない
-        verify(densukeMemberMappingRepository, never()).findByDensukeUrlIdAndPlayerId(any(), any());
-        assertThat(densukeWriteService.getStatus(1L).getPendingCount()).isEqualTo(0);
+            densukeWriteService.writeToDensuke();
+
+            // BYEのみdirtyの場合、書き込み処理に進まない
+            verify(densukeMemberMappingRepository, never()).findByDensukeUrlIdAndPlayerId(any(), any());
+            assertThat(densukeWriteService.getStatus(1L).getPendingCount()).isEqualTo(0);
+        }
     }
 
     // ----------------------------------------------------------------

--- a/karuta-tracker/src/test/java/com/karuta/matchtracker/service/WaitlistPromotionServiceTest.java
+++ b/karuta-tracker/src/test/java/com/karuta/matchtracker/service/WaitlistPromotionServiceTest.java
@@ -654,7 +654,7 @@ class WaitlistPromotionServiceTest {
                     .id(1L).sessionId(100L).playerId(10L).matchNumber(1)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1).build();
             PracticeSession session = PracticeSession.builder()
-                    .id(100L).sessionDate(LocalDate.of(2026, 5, 1)).build();
+                    .id(100L).sessionDate(JstDateTimeUtil.today().plusDays(7)).build();
             PracticeParticipant waitlisted = PracticeParticipant.builder()
                     .id(2L).sessionId(100L).playerId(20L).matchNumber(1)
                     .status(ParticipantStatus.WAITLISTED).waitlistNumber(2).build();
@@ -671,7 +671,7 @@ class WaitlistPromotionServiceTest {
                             100L, 1, ParticipantStatus.WAITLISTED))
                     .thenReturn(Optional.of(waitlisted));
             when(lotteryDeadlineHelper.calculateOfferDeadline(any()))
-                    .thenReturn(java.time.LocalDateTime.of(2026, 5, 10, 18, 0));
+                    .thenReturn(JstDateTimeUtil.now().plusDays(7));
 
             service.expireOffer(offered);
 
@@ -1061,16 +1061,16 @@ class WaitlistPromotionServiceTest {
             PracticeParticipant p1 = PracticeParticipant.builder()
                     .id(1L).sessionId(100L).playerId(10L).matchNumber(1)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2026, 5, 10, 18, 0))
+                    .offerDeadline(JstDateTimeUtil.now().plusDays(7))
                     .build();
             PracticeParticipant p2 = PracticeParticipant.builder()
                     .id(2L).sessionId(100L).playerId(10L).matchNumber(3)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2026, 5, 10, 18, 0))
+                    .offerDeadline(JstDateTimeUtil.now().plusDays(7))
                     .build();
 
             PracticeSession session = PracticeSession.builder().id(100L)
-                    .sessionDate(LocalDate.of(2026, 5, 10)).build();
+                    .sessionDate(JstDateTimeUtil.today().plusDays(7)).build();
             Player triggerPlayer = Player.builder().id(10L).name("テスト選手").build();
 
             when(practiceParticipantRepository.findBySessionIdAndPlayerIdAndStatus(100L, 10L, ParticipantStatus.OFFERED))
@@ -1139,16 +1139,16 @@ class WaitlistPromotionServiceTest {
             PracticeParticipant valid = PracticeParticipant.builder()
                     .id(1L).sessionId(100L).playerId(10L).matchNumber(1)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2026, 5, 10, 18, 0)) // 未来
+                    .offerDeadline(JstDateTimeUtil.now().plusDays(7)) // 未来
                     .build();
             PracticeParticipant expired = PracticeParticipant.builder()
                     .id(2L).sessionId(100L).playerId(10L).matchNumber(3)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2020, 1, 1, 0, 0)) // 過去
+                    .offerDeadline(JstDateTimeUtil.now().minusDays(7)) // 過去
                     .build();
 
             PracticeSession session = PracticeSession.builder().id(100L)
-                    .sessionDate(LocalDate.of(2026, 5, 10)).build();
+                    .sessionDate(JstDateTimeUtil.today().plusDays(7)).build();
 
             when(practiceParticipantRepository.findBySessionIdAndPlayerIdAndStatus(100L, 10L, ParticipantStatus.OFFERED))
                     .thenReturn(List.of(valid, expired));
@@ -1212,7 +1212,7 @@ class WaitlistPromotionServiceTest {
             PracticeParticipant participant = PracticeParticipant.builder()
                     .id(1L).sessionId(100L).playerId(10L).matchNumber(1)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2026, 5, 10, 18, 0))
+                    .offerDeadline(JstDateTimeUtil.now().plusDays(7))
                     .build();
 
             PracticeParticipant remaining = PracticeParticipant.builder()
@@ -1244,7 +1244,7 @@ class WaitlistPromotionServiceTest {
             PracticeParticipant participant = PracticeParticipant.builder()
                     .id(1L).sessionId(100L).playerId(10L).matchNumber(1)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2026, 5, 10, 18, 0))
+                    .offerDeadline(JstDateTimeUtil.now().plusDays(7))
                     .build();
             PracticeSession session = PracticeSession.builder().id(100L).build();
             Player triggerPlayer = Player.builder().id(10L).name("テスト選手").build();
@@ -1271,7 +1271,7 @@ class WaitlistPromotionServiceTest {
             PracticeParticipant participant = PracticeParticipant.builder()
                     .id(1L).sessionId(100L).playerId(10L).matchNumber(1)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2026, 5, 10, 18, 0))
+                    .offerDeadline(JstDateTimeUtil.now().plusDays(7))
                     .build();
 
             when(practiceParticipantRepository.findById(1L)).thenReturn(Optional.of(participant));
@@ -1302,13 +1302,13 @@ class WaitlistPromotionServiceTest {
             PracticeParticipant participant = PracticeParticipant.builder()
                     .id(1L).sessionId(100L).playerId(10L).matchNumber(1)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2026, 5, 10, 18, 0))
+                    .offerDeadline(JstDateTimeUtil.now().plusDays(7))
                     .build();
             PracticeParticipant nextWaitlisted = PracticeParticipant.builder()
                     .id(5L).sessionId(100L).playerId(20L).matchNumber(1)
                     .status(ParticipantStatus.WAITLISTED).waitlistNumber(2).build();
             PracticeSession session = PracticeSession.builder().id(100L)
-                    .sessionDate(LocalDate.of(2026, 5, 10)).build();
+                    .sessionDate(JstDateTimeUtil.today().plusDays(7)).build();
 
             when(practiceParticipantRepository.findById(1L)).thenReturn(Optional.of(participant));
             when(practiceSessionRepository.findById(100L)).thenReturn(Optional.of(session));
@@ -1323,7 +1323,7 @@ class WaitlistPromotionServiceTest {
                             100L, 1, ParticipantStatus.WAITLISTED))
                     .thenReturn(Optional.of(nextWaitlisted));
             when(lotteryDeadlineHelper.calculateOfferDeadline(any()))
-                    .thenReturn(java.time.LocalDateTime.of(2026, 5, 10, 18, 0));
+                    .thenReturn(JstDateTimeUtil.now().plusDays(7));
             when(playerRepository.findById(10L)).thenReturn(Optional.of(Player.builder().id(10L).name("テスト選手").build()));
 
             service.respondToOffer(1L, false);
@@ -1349,7 +1349,7 @@ class WaitlistPromotionServiceTest {
                     .id(6L).sessionId(100L).playerId(30L).matchNumber(3)
                     .status(ParticipantStatus.WAITLISTED).waitlistNumber(2).build();
             PracticeSession session = PracticeSession.builder().id(100L)
-                    .sessionDate(LocalDate.of(2026, 5, 10)).build();
+                    .sessionDate(JstDateTimeUtil.today().plusDays(7)).build();
 
             when(practiceParticipantRepository.findBySessionIdAndPlayerIdAndStatus(100L, 10L, ParticipantStatus.OFFERED))
                     .thenReturn(List.of(p1, p2));
@@ -1369,7 +1369,7 @@ class WaitlistPromotionServiceTest {
                             100L, 3, ParticipantStatus.WAITLISTED))
                     .thenReturn(Optional.of(nextForMatch3));
             when(lotteryDeadlineHelper.calculateOfferDeadline(any()))
-                    .thenReturn(java.time.LocalDateTime.of(2026, 5, 10, 18, 0));
+                    .thenReturn(JstDateTimeUtil.now().plusDays(7));
             when(playerRepository.findById(10L)).thenReturn(Optional.of(Player.builder().id(10L).name("テスト選手").build()));
 
             service.respondToOfferAll(100L, 10L, false);
@@ -1385,10 +1385,10 @@ class WaitlistPromotionServiceTest {
             PracticeParticipant participant = PracticeParticipant.builder()
                     .id(1L).sessionId(100L).playerId(10L).matchNumber(1)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2026, 5, 10, 18, 0))
+                    .offerDeadline(JstDateTimeUtil.now().plusDays(7))
                     .build();
             PracticeSession session = PracticeSession.builder().id(100L)
-                    .sessionDate(LocalDate.of(2026, 5, 10)).build();
+                    .sessionDate(JstDateTimeUtil.today().plusDays(7)).build();
 
             when(practiceParticipantRepository.findById(1L)).thenReturn(Optional.of(participant));
             when(practiceSessionRepository.findById(100L)).thenReturn(Optional.of(session));
@@ -1418,16 +1418,16 @@ class WaitlistPromotionServiceTest {
             PracticeParticipant valid = PracticeParticipant.builder()
                     .id(1L).sessionId(100L).playerId(10L).matchNumber(1)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2026, 5, 10, 18, 0)) // 未来
+                    .offerDeadline(JstDateTimeUtil.now().plusDays(7)) // 未来
                     .build();
             PracticeParticipant expired = PracticeParticipant.builder()
                     .id(2L).sessionId(100L).playerId(10L).matchNumber(3)
                     .status(ParticipantStatus.OFFERED).waitlistNumber(1)
-                    .offerDeadline(java.time.LocalDateTime.of(2020, 1, 1, 0, 0)) // 過去
+                    .offerDeadline(JstDateTimeUtil.now().minusDays(7)) // 過去
                     .build();
 
             PracticeSession session = PracticeSession.builder().id(100L)
-                    .sessionDate(LocalDate.of(2026, 5, 10)).build();
+                    .sessionDate(JstDateTimeUtil.today().plusDays(7)).build();
             Player triggerPlayer = Player.builder().id(10L).name("テスト選手").build();
 
             when(practiceParticipantRepository.findBySessionIdAndPlayerIdAndStatus(100L, 10L, ParticipantStatus.OFFERED))
@@ -1488,10 +1488,10 @@ class WaitlistPromotionServiceTest {
                         100L, 1, ParticipantStatus.WAITLISTED))
                 .thenReturn(Optional.of(waitlisted));
         when(lotteryDeadlineHelper.calculateOfferDeadline(any()))
-                .thenReturn(java.time.LocalDateTime.of(2026, 5, 10, 18, 0));
+                .thenReturn(JstDateTimeUtil.now().plusDays(7));
 
         Optional<PracticeParticipant> result = service.promoteNextWaitlisted(
-                100L, 1, LocalDate.of(2026, 5, 5));
+                100L, 1, JstDateTimeUtil.today().plusDays(7));
 
         assertThat(result).isPresent();
         assertThat(result.get().getStatus()).isEqualTo(ParticipantStatus.OFFERED);
@@ -1515,7 +1515,7 @@ class WaitlistPromotionServiceTest {
                     .id(5L).sessionId(100L).playerId(20L).matchNumber(1)
                     .status(ParticipantStatus.WAITLISTED).waitlistNumber(2).build();
             PracticeSession session = PracticeSession.builder().id(100L)
-                    .sessionDate(LocalDate.of(2026, 5, 10)).build();
+                    .sessionDate(JstDateTimeUtil.today().plusDays(7)).build();
 
             when(practiceSessionRepository.findById(100L)).thenReturn(Optional.of(session));
             when(practiceParticipantRepository
@@ -1527,7 +1527,7 @@ class WaitlistPromotionServiceTest {
                             100L, 1, ParticipantStatus.WAITLISTED))
                     .thenReturn(Optional.of(nextWaitlisted));
             when(lotteryDeadlineHelper.calculateOfferDeadline(any()))
-                    .thenReturn(java.time.LocalDateTime.of(2026, 5, 10, 18, 0));
+                    .thenReturn(JstDateTimeUtil.now().plusDays(7));
 
             ExpireOfferResult result = service.expireOfferSuppressed(participant);
 

